src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -2234,6 +2234,7 @@ public enum SignatureCallingConvention : byte
         ThisCall = (byte)3,
         FastCall = (byte)4,
         VarArgs = (byte)5,
+        Unmanaged = (byte)9,
```

Comment:
Apologies if we've already bikeshedded this, but do we want `Unmanaged` here, or something like `UnmanagedDefault` or `PlatformDefault`? Technically 4/6 of the existing conventions are unmanaged, so just saying `Unmanaged` might be confusing?

Reply:
That's a good question for API review that did not happened yet for this one.

Personally, I am fine with just `Unmanaged`. A reasonable name won't be able to capture the full semantics of this (`UnmanagedPlatformDefaultWithOptionalModOptOverridesDoNotUseForCdeclStdCallFastCall`), so it seems reasonable to just use a name that looks good.

Reply:
>That's a good question for API review that did not happened yet for this one.

Indeed. I will wait for Thursday API review.

Reply:
Handle in API review.